frontend/app/services/personal-information-service.server.ts

```diff
@@ -185,5 +151,48 @@ function createPersonalInformationService() {
     throw new Error(`Failed to fetch data. address: ${response.status}, Status Text: ${response.statusText}`);
   }
 
+  function toPersonalInformation(personalnformationDto: PersonalInformationApiResponse): PersonalInfo {
+    const addressList = personalnformationDto.Client.PersonContactInformation?.Address;
+    const homeAddressList = addressList?.filter((address) => address.AddressCategoryCode.ReferenceDataName == 'Home');
+    const mailingAddressList = addressList?.filter((address) => address.AddressCategoryCode.ReferenceDataName == 'MAiling');
+
+    return {
+      clientInfoList: personalnformationDto.Client.ClientIdentification
+        ? personalnformationDto.Client.ClientIdentification.map((clientInfoDto) => ({
+            id: clientInfoDto.IdentificationID,
+            category: clientInfoDto.IdentificationCategoryText,
+          }))
+        : [],
+      firstName: personalnformationDto.Client.PersonName ? personalnformationDto.Client.PersonName.at(0)?.PersonGivenName?.at(0) : '',
+      lastName: personalnformationDto.Client.PersonName ? personalnformationDto.Client.PersonName.at(0)?.PersonSurName?.at(0) : '',
+      emailAddress: personalnformationDto.Client.PersonContactInformation ? personalnformationDto.Client.PersonContactInformation.EmailAddress.EmailAddressID : '',
+      phoneNumber: personalnformationDto.Client.PersonContactInformation ? personalnformationDto.Client.PersonContactInformation.TelephoneNumber.FullTelephoneNumber : '',
+      homeAddress: homeAddressList
+        ?.map((aHomeAddress) => ({
+          categoryCode: aHomeAddress.AddressCategoryCode.ReferenceDataName ? aHomeAddress.AddressCategoryCode.ReferenceDataName : '',
+          streetName: aHomeAddress.AddressStreet.StreetName ? aHomeAddress.AddressStreet.StreetName : ' ',
+          cityName: aHomeAddress.AddressCityName ? aHomeAddress.AddressCityName : '',
+          provinceName: aHomeAddress.AddressProvince.ProvinceName ? aHomeAddress.AddressProvince.ProvinceName : '',
+          countryName: aHomeAddress.AddressCountry.CountryName ? aHomeAddress.AddressCountry.CountryName : '',
+          postalCode: aHomeAddress.AddressPostalCode ? aHomeAddress.AddressPostalCode : '',
```

Comment:
I see that the `personalInformationApiResponseSchema` has all these fields as mandatory, so I feel we shouldn't need to check for the existence of the field and default it to an empty string in our application model. Although I feel that some of these fields should be optional like `provinceName` and `postalCode`.